README.md

```diff
@@ -2,55 +2,40 @@
 [![npm version][2]][3] [![build status][4]][5] [![test coverage][6]][7]
 [![downloads][8]][9] [![js-standard-style][10]][11]
 
-Fast lisp-like router for streaming servers.
+Performant radix-trie router for streaming servers.
```

Comment:
can you link to [something like this](https://en.wikipedia.org/wiki/Radix_tree) maybe? i defo had to google this